### PR TITLE
Updated Android Release Notes for April 2021.

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: releases/2021-04/android.md [hunk 1]
```diff
@@ -49,10 +143,178 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+#### 1.0.0-beta.4 ([Changelog](https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/core/azure-core-http-okhttp/CHANGELOG.md#100-beta4-2021-03-18))
+
+##### Breaking Changes
+
+- Initial release.
+
+### Azure Core Http OkHttp
+
+#### 1.0.0-beta.5 ([Changelog](https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/core/azure-core-http-okhttp/CHANGELOG.md#100-beta5-2021-03-26))
+
```
Comment (KarishmaGhiya) [inline releases/2021-04/android.md:255]: Do you want to mention what changed?

File: releases/2021-04/android.md [hunk 2]
```diff
@@ -49,10 +143,178 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+#### 1.0.0-beta.4 ([Changelog](https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/core/azure-core-http-okhttp/CHANGELOG.md#100-beta4-2021-03-18))
+
+##### Breaking Changes
+
+- Initial release.
```
Comment (KarishmaGhiya) [inline releases/2021-04/android.md:260]: Do you want to mention what changed? Are there any developer imapcting changes?

File: releases/2021-04/android.md [hunk 3]
```diff
@@ -49,10 +143,178 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+##### Breaking Changes
+
+- Removed the `azure-core.properties` file.
+
+#### 1.0.0-beta.4 ([Changelog](https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/core/azure-core-credential/CHANGELOG.md#100-beta4-2021-03-18))
+
+##### Breaking Changes
+
+- Initial release.
```
Comment (KarishmaGhiya) [inline releases/2021-04/android.md:216]: Do you want to mention what changed? Are there any developer impacting changes? How does the split up affect customers?

**Review by KarishmaGhiya — APPROVED** [review]

File: releases/2021-04/android.md (hunk 1 above)
Comment (vcolin7) [reply releases/2021-04/android.md:255]: No changes were made to the package in this release, we just did it to keep them all on track with the same number.

File: releases/2021-04/android.md (hunk 2 above)
Comment (vcolin7) [reply releases/2021-04/android.md:260]: Sure, we can do that.

**Pull request merged**